fr.json

```diff
@@ -61,8 +61,8 @@
     "dmst": "DMSt",
     "free": "Free",
     "nsfw": "NSFW",
-    "out": "Out",
-    "out_return": "O&R",
+    "out": "Ligne droite",
```

Comment:
We have not enough space for such a long word in the UI. Should be < 8 Characters.
Do you have maybe an abbreviation?